include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -928,6 +928,8 @@ struct SessionOptionsImpl : ConstSessionOptionsImpl<T> {
 
   SessionOptionsImpl& SetExecutionMode(ExecutionMode execution_mode);  ///< Wraps OrtApi::SetSessionExecutionMode
 
+  SessionOptionsImpl& SetLoadCancellationFlag(bool value);  ///< Wraps OrtApi::SessionOptionsSetLoadCancellationFlag
```

Comment:
what is the use case of `value == false`?

Reply:
Undo cancellation if the user changes its mind before it is cancelled. In general, I have done this to reduce an existing proliferation of the interfaces that are sometimes present in pairs in our API. (set/unet).